google/cloud/firestore_v1/base_collection.py

```diff
@@ -507,6 +507,30 @@ def count(self, alias=None):
         """
         return self._aggregation_query().count(alias=alias)
 
+    def sum(self, field_ref: str, alias=None):
```

Comment:
does FieldPath need to be appended here as well?

Reply:
good catch! Fixed

(The type checking system should have caught this, but it looks like it's not running properly. I'll have to fix that in a separate PR)

Reply:
sgtm, please file an issue for tracking!

Reply:
I have a bug for it here: https://github.com/googleapis/python-firestore/issues/773